[Bug] Can't resolve 'global/window' error after updating to Storybook v7

### Describe the bug

I import the plugin in my `preview.js` with a simple import:

```
import '@storybook/addon-console';
```

It works fine for Storybook v6, but I'm getting an error after updating to v7.

### Expected behavior

The addon works with Storybook v7.

### Screenshots and/or logs

```
ERROR in ./node_modules/@storybook/addon-console/dist/index.js 8:37-61
Module not found: Error: Can't resolve 'global/window'
```

### Environment

 - Storybook version: v7.0.0-rc.7
 - OS: macOS
 - Node.js version: v18.15.0
 - NPM version: v8.19.4
 - Browser (if applicable): Chrome

--- a/package.json
+++ b/package.json
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "start": "start-storybook -p 9009 --ci",
-    "prepare": "package-prepare && npx typescript 'src/index.js' --declaration --allowJs --emitDeclarationOnly --outDir dist",
+    "prepare": "package-prepare && yarn build",
+    "build": "tsc \"src/index.js\" --declaration --allowJs --emitDeclarationOnly --outDir dist",
     "build:tests": "rm -rdf .jest-dist && babel ./src -d ./.jest-dist -s",
     "build:readme.md": "jsdoc2md -t docs/readme.hbs -c docs/jsdoc.config.json src/*.js  > README.md; echo",
     "build:readme": "jsdoc2md -t docs/readme.hbs -c docs/jsdoc.config.json src/*.js  > README; echo",
@@ -93,6 +94,7 @@
     "react-dom": "^16.4.0",
     "react-scripts": "2.0.5",
     "react-test-renderer": "^16.4.0",
+    "typescript": "^5.0.3",
     "url-loader": "^2.1.0"
   },
   "peerDependencies": {

--- a/package.json
+++ b/package.json
@@ -4,7 +4,7 @@
   "author": "Oleg Proskurin (https://github.com/UsulPro)",
   "description": "Show console output like logs, errors, and warnings in the Storybook",
   "dependencies": {
-    "global": "^4.3.2"
+    "@storybook/global": "^5.0.0"
   },
   "scripts": {
     "start": "start-storybook -p 9009 --ci",

--- a/src/index.js
+++ b/src/index.js
@@ -5,7 +5,7 @@
  *
  */
 
-import window from 'global/window';
+import { global } from '@storybook/global';
 import { action, configureActions } from '@storybook/addon-actions';
 
 import { reactStory } from './react-decorator';
@@ -90,7 +90,7 @@ function setScope(options) {
     if (toConsole.length) cLogger.error(...toConsole);
   };
 
-  window.onerror = (...args) => {
+  global.onerror = (...args) => {
     const toPanel = shouldDisplay([args[0]], panelExclude, panelInclude);
     const toConsole = shouldDisplay([args[0]], consoleExclude, consoleInclude);
     if (toPanel.length) aLogger.error(...args);
@@ -155,12 +155,12 @@ export function setConsoleOptions(optionsOrFn) {
 }
 
 function handleStoryLogs() {
-  switch (window.STORYBOOK_ENV) {
+  switch (global.STORYBOOK_ENV) {
     case 'react':
       return reactStory;
     default:
       logger.warn(
-        `Warning! withConsole doesn't support @storybook/${window.STORYBOOK_ENV}. Use setConsoleOptions instead`
+        `Warning! withConsole doesn't support @storybook/${global.STORYBOOK_ENV}. Use setConsoleOptions instead`
       );
       return story => story;
   }
@@ -170,10 +170,10 @@ function addConsole(storyFn, context, consoleOptions) {
   const prevOptions = { ...currentOptions };
   const logNames = context
     ? {
-        log: `${context.kind}/${context.story}`,
-        warn: `${context.kind}/${context.story}/warn`,
-        error: `${context.kind}/${context.story}/error`,
-      }
+      log: `${context.kind}/${context.story}`,
+      warn: `${context.kind}/${context.story}/warn`,
+      error: `${context.kind}/${context.story}/error`,
+    }
     : {};
 
   const options = {

